src/Methods/ConfigurationInteraction/CASCI/SparseHamiltonian.jl

```diff
@@ -40,8 +41,8 @@ function CASCI{T}(Alg::SparseHamiltonian) where T <: AbstractFloat
     end
 
     s = 1:(frozen+active)
-    h = T.(Fermi.Integrals.transform_fock(aoint.T+aoint.V, refwfn.C[:,s], refwfn.C[:,s]))
-    V = T.(Fermi.Integrals.transform_eri(aoint.ERI, refwfn.C[:,s], refwfn.C[:,s], refwfn.C[:,s], refwfn.C[:,s]))
+    h = T.(Fermi.Integrals.transform_fock(ints["T"] + ints["V"], ints.C["C"][:,s], ints.C["C"][:,s]))
```

Comment:
`ints.C["C"][:,s]` seems confusing...
ints.C are the orbitals? Why ["C"] ?
Also, should transform_fock be part of ints?